config/tech-docs.yml

```diff
@@ -11,6 +11,8 @@ phase: INTERNAL
 header_links:
   Documentation: /
   GitHub: https://github.com/alphagov/reliability-engineering
+footer_links:
+  Reliability Engineering Internal Docs Accessibility: /accessibility.html
```

Comment:
This is a bit of a mouthful of a link :smile: ... is that intentional?

```suggestion
  Accessibility: /accessibility.html
```

Reply:
I agree it's long, but this is the format used elsewhere. I'll query with the other tech writers and update!

Reply:
It is long, but as Sabrina says this is consistent with other tech docs statements. We try to ensure it is explicit that these statements are for the tech content itself rather than the general website.

Reply:
I think you could just use 'Accessibility'. I've done that where the link text would have otherwise been super long.

Reply:
I don't want to block this PR, it's not an issue ...  it can easily be changed in the future, just wasn't sure if intentional ... merge away! :+1: